lib/cli/run.js

```diff
@@ -258,6 +258,16 @@ exports.builder = yargs =>
         }
       });
 
+      types.boolean.concat(types.string, types.number).forEach(opt => {
+        if (Array.isArray(argv[opt])) {
+          throw createInvalidArgumentValueError(
+            `Argument "${opt}" can only be provided once`,
+            opt,
+            argv[opt]
+          );
+        }
+      });
+
```

Comment:
Disagree. If `Array`, convert back to original type using last value of `argv[opt]`
(i.e., pretend `yargs-parser.duplicate-arguments-array = false` for _just_ these types).